verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -92,6 +93,8 @@ verible-verilog-kythe-extractor files...)");
   const auto args = verible::InitCommandLine(usage, &argc, &argv);
 
   int exit_status = 0;
+  std::vector<verilog::kythe::IndexingFactNode> indexing_facts_trees;
```

Comment:
Ok for now, but in follow-ups consider managing multiple files' facts trees inside a class/structure intended for multi-file management.  In the long run, single-file operation should be a degenerate case of multi-file operation.

Reply:
David, can you expand on your comment? I want to make sure you and I don't send different messages. This looks OK to me.

CST --> Middle layer works file-by-file. Middle layer --> Kythe takes all files at once.

Maybe it would help to encapsulate all in a single class?

Reply:
For example, encapsulation within a single structure/class would be a good place to establish a mapping between root-level symbol definitions and their files of origin.  Why search through each file's root scope when a top-level map could give you the same information more efficiently?  That is to say that the root-level symbol table (or equivalent) should handle multiple files.